### DISCUSSION TITLE
Why is there such a big difference between the compile time and the done time?

### DISCUSSION THREAD

**Discussion by zhouxinyong** [web-infra-dev/rsbuild#4294] [Q&A]
In my project CI:
compile time is 10s
![image](https://github.com/user-attachments/assets/9b5e47cb-77ef-41c0-9b39-16ca501ac87f)
done time is 44s
![image](https://github.com/user-attachments/assets/05e03d3c-9919-4284-941a-c51c33a33e30)

Is there any way to shorten this time?

**Comment by chenjiahan** [comment]
During production build, Rsbuild will register the SWC minify plugin, which may take some time.

Can you try to disable minify and compare the build time?

https://rsbuild.dev/config/output/minify

**Reply by zhouxinyong** [reply]
@chenjiahan 
when i disable minify
build time：7s
![image](https://github.com/user-attachments/assets/6b325f27-68eb-43c7-8fee-918e0738cad5)
done time: 21s
![image](https://github.com/user-attachments/assets/1f66bdce-d819-4480-bce5-9e0627ed30fb)

the done time reduce 20s.

but in prod build,how to reduce this time?

**Reply by chenjiahan** [reply]
It seems that this is caused by SWC's minify performance. The SWC team is working on optimization and I think there will be a significant improvement soon.
